CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes -->
 
+## [v0.96.0-sumo-1]
+
+### Released 2024-03-20
+
+### Added
+
+- feat(opamp): Support adding and validating more receivers and extensions in the source templates [#1509]
+
+### Fixed
+
+- - fix(windows): Fixed startup problem when running as a Windows service [#1512]
```

Comment:
There is a double `-` here, probably why the md validation is failing?

Reply:
it fails because this link doesn't exist yet
https://github.com/SumoLogic/sumologic-otel-collector/releases/v0.96.0-sumo-1

which is expected, I'll remove the extra `-` though